ddtrace/monkey.py

```diff
@@ -79,7 +79,23 @@ def patch(raise_errors=True, **patch_modules):
     modules = [m for (m, should_patch) in patch_modules.items() if should_patch]
     count = 0
     for module in modules:
-        patched = patch_module(module, raise_errors=raise_errors)
+        # TODO: this is a temporary hack until we shift to using
+        # post-import hooks for everything.
+        if module == 'celery':
+            # if patch celery via post-import hooks
+            from wrapt.importer import when_imported
+
+            @when_imported('celery')
+            def patch_celery(hook):
+                from ddtrace.contrib.celery import patch
+                patch()
+
+            # manually add celery to patched modules and increment count
+            _PATCHED_MODULES.add(module)
+            count += 1
+            patched = True
```

Comment:
reading the code I think `count` is incremented twice :)

Reply:
ugh, yes, good catch! we really need to change how we patch so that this dirty hack can go away very soon...